From 0f2d92fa68dda91d13dc62bedd83c5cc1d705d0f Mon Sep 17 00:00:00 2001
From: Stefan Riedmueller <s.riedmueller@phytec.de>
Date: Fri, 21 Jan 2022 15:25:04 +0100
Subject: [PATCH 1/2] Revert "MMFMWK-8918 [dmabufheaps] enable dmabuf heaps
 memory allocator."

This reverts commit 6dd8dc7566a76354e3283dc4e2dcecae817bf20e.

Signed-off-by: Stefan Riedmueller <s.riedmueller@phytec.de>
---
 gst-libs/gst/allocators/gstdmabuf.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/gst-libs/gst/allocators/gstdmabuf.c b/gst-libs/gst/allocators/gstdmabuf.c
index 5c482f9e15fe..a91cbcce15ca 100644
--- a/gst-libs/gst/allocators/gstdmabuf.c
+++ b/gst-libs/gst/allocators/gstdmabuf.c
@@ -70,9 +70,9 @@ gst_dmabuf_mem_map (GstMemory * gmem, GstMapInfo * info, gsize maxsize)
 #ifdef HAVE_LINUX_DMA_BUF_H
   if (ret) {
     struct dma_buf_phys dma_phys;
-    /* if (ioctl (gst_fd_memory_get_fd (gmem), DMA_BUF_IOCTL_SYNC, &sync) < 0)
-       GST_WARNING_OBJECT (allocator, "Failed to synchronize DMABuf: %s (%i)",
-       g_strerror (errno), errno); */
+    if (ioctl (gst_fd_memory_get_fd (gmem), DMA_BUF_IOCTL_SYNC, &sync) < 0)
+      GST_WARNING_OBJECT (allocator, "Failed to synchronize DMABuf: %s (%i)",
+          g_strerror (errno), errno);
     ioctl (gst_fd_memory_get_fd (gmem), DMA_BUF_IOCTL_PHYS, &dma_phys);
   }
 #endif
@@ -94,9 +94,9 @@ gst_dmabuf_mem_unmap (GstMemory * gmem, GstMapInfo * info)
   if (info->flags & GST_MAP_WRITE)
     sync.flags |= DMA_BUF_SYNC_WRITE;
 
-  /* if (ioctl (gst_fd_memory_get_fd (gmem), DMA_BUF_IOCTL_SYNC, &sync) < 0)
-     GST_WARNING_OBJECT (allocator, "Failed to synchronize DMABuf: %s (%i)",
-     g_strerror (errno), errno); */
+  if (ioctl (gst_fd_memory_get_fd (gmem), DMA_BUF_IOCTL_SYNC, &sync) < 0)
+    GST_WARNING_OBJECT (allocator, "Failed to synchronize DMABuf: %s (%i)",
+        g_strerror (errno), errno);
   ioctl (gst_fd_memory_get_fd (gmem), DMA_BUF_IOCTL_PHYS, &dma_phys);
 #else
   GST_WARNING_OBJECT (allocator, "Using DMABuf without synchronization.");
-- 
2.25.1

